webrender/src/resource_cache.rs

```diff
@@ -554,8 +554,17 @@ impl ResourceCache {
                 }
                 _ => {}
             }
+        }
 
-            match update {
+        drain_filter(
+            updates,
+            |update: &mut ResourceUpdate| match *update {
```

Comment:
I thought so too, then I was battling some build errors and ended up putting that and thought it was what helped. Turns out this wasn't what fixed my trouble since it indeed compiles when I remove the argument type now.